### DISCUSSION TITLE
Has BloomFilter and Json type key supported slot migrations

### DISCUSSION THREAD

**Discussion by jihuayu** [apache/kvrocks#1934] [General]
When I was reading the code in [slot_migrate.cc](https://github.com/jihuayu/incubator-kvrocks/blob/5e60dc255dde6da3ef4aa9913acfb9741b38fbb4/src/cluster/slot_migrate.cc), I noticed that slot migration seems to be not working for the new types `kRedisBloomFilter` and `kRedisJson`. 

I wanted to ask if my understanding is incorrect. What happens now when slot migration involves these two types?

**Something related:**
- https://github.com/apache/kvrocks/issues/1805
- https://github.com/apache/kvrocks/issues/1696

**Comment by git-hulk** [comment]
@jihuayu Yes, you're right. I'm thinking about how to make those migration codes easier to maintain. It's too hard for contributors to take care of this when adding the new command.
